recipes-bsp/tegra-binaries/tegra-helper-scripts/initrd-flash.sh

```diff
@@ -243,20 +243,26 @@ sign_binaries() {
     fi
     if [ "$CHIPID" = "0x18" ]; then
 	"$here/tegra-signimage-helper" --chip "$CHIPID" --nosplit --type kernel initrd-flash.img
+	if [ ! -z "${INITRD_FLASHER_DTBFILE}" ]; then
+	    "$here/tegra-signimage-helper" --chip "$CHIPID" --nosplit --type kernel-dtb ${INITRD_FLASHER_DTBFILE}
+	fi
     fi
     . ./boardvars.sh
     return 0
 }
 
 prepare_for_rcm_boot() {
+    if [ -z "${INITRD_FLASHER_DTBFILE}" ]; then
```

Comment:
Redefining the global variable as local here could be confusing later.  Could you do something like
```
local flasherdtb="${INITRD_FLASHER_DTBFILE:-kernel_${DTBFILE}}"
```
instead, and then use `$flasherdtb` below?

Reply:
Should be resolved now.